src/game_monster.ts

```diff
@@ -465,6 +471,13 @@ export class GameMonster extends GameCard {
     return currentMelee;
   }
 
+  resurrect() {
+    this.health = 1;
+    this.armor = this.startingArmor;
+    if (this.hadDivineShield) { this.addAbility(Ability.DIVINE_SHIELD) }
```

Comment:
missing semi colon

Move this to it's ownline
```
if (this.hadDivineShield) { 
  this.addAbility(Ability.DIVINE_SHIELD);
}
```

Reply:
Done